### PR TITLE
Set empty title for material design icons

### PR DESCRIPTION
**Pull request by PVince81**
### Description

The vue material design icons library sets a default title to the icon
components. This fix sets this to empty to prevent the browser tooltip
to appear.

### Before
image

### After
No more tooltip

### Context
Discovered here: https://github.com/nextcloud/spreed/pull/4621#discussion_r525302207

### CODE REVIEW

File: src/components/CallView/Grid/Grid.vue [hunk 1]
```diff
@@ -41,7 +43,9 @@
 					<button v-if="hasPreviousPage && gridWidth > 0 && isStripe && showVideoOverlay"
 						class="grid-navigation grid-navigation__previous"
 						@click="handleClickPrevious">
-						<ChevronLeft decorative
+						<ChevronLeft
+							decorative
```
Comment (danxuliu) [inline src/components/CallView/Grid/Grid.vue:47]: As [the icon will be now hidden from screen readers](https://github.com/robcresswell/vue-material-design-icons#props) an `aria-label` should be added to the parent button. Or a tooltip. But there are other cases in which this should be done too, so I think it would be better to solve it in another pull request (maybe as part of #4621 once rebased?)

**Review by danxuliu — APPROVED** [review]
Tested and works :+1: 

A curious detail: in order to show the icon tooltip I had to hover exactly on the strokes in the icon; just hovering on the icon but not exactly on a black stroke did not show anything.

File: src/components/CallView/Grid/Grid.vue (hunk 1 above)
Comment (PVince81) [reply src/components/CallView/Grid/Grid.vue:47]: I'll do it in this PR right away.

Also I think aria-label needs to not describe the icon like the title did (what does a "chevron icon" do anyway) but the action that the button does.

**Comment by PVince81** [comment]
Now thinking, not all icons are tab-abble, so not sure if they need an aria-label.
We might think about this as part of improved keyboard navigation.
I've added a note here about this: https://github.com/nextcloud/spreed/issues/4647#issuecomment-730181674

**Comment by PVince81** [comment]
> I had to hover exactly on the strokes in the icon

I tried adding a v-tooltip for some buttons and it seems that whenever an icon is inside a "button", the tooltip only appears when you hover in the space between the button border and the icon pixels. Hovering on the icon pixels makes the tooltip disappear... Go figure... Entering can of worms territory :-/

File: src/components/CallView/Grid/Grid.vue (hunk 1 above)
Comment (danxuliu) [reply src/components/CallView/Grid/Grid.vue:47]: > Also I think aria-label needs to not describe the icon like the title did (what does a "chevron icon" do anyway) but the action that the button does.

Oh, totally. But my point was that without `decorative` a screen reader would say (as far as I understand, I have not tried it) _Chevron Icon Left_ which, even if it is not the best description, at least says _icon left_, but with `decorative` and without aria-label or tooltip on the button then the screen reader would only say _button_, which is even less descriptive.

**Comment by danxuliu** [comment]
> Now thinking, not all icons are tab-abble, so not sure if they need an aria-label.

I think that even in that case the icon should be readable if it is used to convey information like _in the call_.

> I tried adding a v-tooltip for some buttons and it seems that whenever an icon is inside a "button", the tooltip only appears when you hover in the space between the button border and the icon pixels. Hovering on the icon pixels makes the tooltip disappear... Go figure... Entering can of worms territory :-/

Maybe the icon title overrides the tooltip, even if the title is empty? I have no idea, it is just a wild guess :-)

**Comment by PVince81** [comment]
@danxuliu I've added aria labels for the grid buttons which were missing some.
I didn't add a tooltip there as it felt a bit redundant.

> I think that even in that case the icon should be readable if it is used to convey information like in the call.

Ok, I'll do that in the tooltip PR then: https://github.com/nextcloud/spreed/pull/4621

File: src/components/CallView/Grid/Grid.vue (hunk 1 above)
Comment (PVince81) [reply src/components/CallView/Grid/Grid.vue:49]: indeed

**Review by ma12-co — APPROVED** [review]
